extensions/amp-list/0.1/amp-list.js

```diff
@@ -145,6 +133,11 @@ export class AmpList extends AMP.BaseElement {
     }
   }
 
+  /** @override */
```

Comment:
a bit of comment to clarify what's happening here, something along the lines of `amp-list reuses the loading indicator when list is fetched again via bind mutation or refresh action`